cohortextractor/patients.py

```diff
@@ -753,6 +755,60 @@ def registered_practice_as_of(
                     },
                 },
             )
+
+
+        ### Cluster RCTs
+
+        Clustered Randomised Control Trials (Cluster RCTs) that are clustered at practice level
+        are available. There is special syntax that signifies RCT. This is `rct__trialname__propertyname`
+        for example, if there was a cluster RCT called germdefence which has a property called
+        deprivation_pctile, a variable can be created by:
+
+            germdefence_intervention=patients.registered_practice_as_of(
```

Comment:
probably would call this variable germdefence_practice_deprivation_pctile or similar